content/docs/command-reference/remote/modify.md

```diff
@@ -35,6 +35,9 @@ manual editing could be used to change the configuration.
 - `-u`, `--unset` - delete configuration value for the given config `option`.
   Don't provide a `value` when employing this flag.
 
+- `--jobs` - number of workers started. Accepts positive integers. The default
+  value is `4 * cpu_count()`.
+
```

Comment:
Wrong section, it should be the one below "Available parameters for all remotes"

Reply:
EDIT: ah, I see. This is the flag that you are adding here. Need to add `jobs` as a config option to "Available parameters for all remotes" below too

Reply:
There is no such option though, see https://github.com/iterative/dvc.org/pull/1941#discussion_r523069333